source/Interfaces/NetWorth/client/ClientInterface.hpp

```diff
@@ -75,34 +79,38 @@ namespace netWorth{
                                 const cse491::item_map_t & item_set,
                                 const cse491::agent_map_t & agent_set) override
             {
-                // Receive and draw map
-                sf::Packet send_pkt, recv_pkt;
-                std::string map;
-
-                ReceivePacket(recv_pkt, m_ip, m_port);
-                ProcessPacket(recv_pkt);
+                sf::Packet send_pkt;
 
                 // grab action ID from MainInterface
               std::uint32_t action_id = i_2D::MainInterface::SelectAction(grid, type_options,
                                                                      item_set, agent_set);
-                std::cout << action_id << std::endl;
+                //std::cout << action_id << std::endl;
```

Comment:
This is super minor, but as we move to the final deadline, feel free to cut lines like this instead of commenting. Old versions are always available on GitHub anyway! 
(I don't think we'd take points off for something like this, unless it's particularly egregious.)